src/control/server/config/server.go

```diff
@@ -470,6 +479,10 @@ func (cfg *Server) Validate(log logging.Logger, hugePageSize int) (err error) {
 	}
 	cfg.AccessPoints = newAPs
 
+	if cfg.Metadata.DevicePath != "" && cfg.Metadata.Path == "" {
```

Comment:
There's no valid use case where there is a device path (e.g. something in /dev) and no Metadata.Path (the path where the user will look for metadata, i.e. the mount path). I didn't want to call it MountPath because the Path may just be an ordinary directory path, in the case where there is no device to mount.